vm/src/stdlib/mod.rs

```diff
@@ -1,6 +1,6 @@
 mod ast;
 mod dis;
-mod json;
+pub mod json;
```

Comment:
Please keep this code private. Having this code unexposed is a good thing, and avoids leaking internals.